tests/conftest.py

```diff
@@ -1,9 +1,27 @@
 import os
 
 import pytest
+from page_analyzer.app import app as test_app
 from page_analyzer.utils.db import connect_db
 
 
+@pytest.fixture
+def app():
+    test_app.config.update(
+        {
+            "TESTING": True,
+        }
+    )
+    test_app.secret_key = "dummy-secret-key"
+
+    return test_app
+
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
```

Comment:
as we're not doing any complicated setup in the `app` fixture, we can unify them into one. wdyt?